src/reducers/__tests__/collapseContext.ts

```diff
@@ -132,3 +132,36 @@ it('merge duplicate children', () => {
     .toMatchObject([{ value: 'a', rank: 0 }, { value: 'c', rank: 0 }])
 
 })
+
+it('after collapse context set cursor to the first visible children.', () => {
+
+  const steps = [
+    newThought('a'),
+    newSubthought('b'),
+    newSubthought('=x'),
+    newThought('c'),
+    cursorBack,
```

Comment:
Tip: I've found that a single `importText` makes for cleaner tests than several `newThought` or `newSubthought` calls. This is acceptable though.